planner/core/common_plans.go

```diff
@@ -661,6 +695,7 @@ type Explain struct {
 
 	Rows        [][]string
 	ExplainRows [][]string
+	JSON        JSONSlice
```

Comment:
Not a big issue, but it looks like we don't need to add a new field here, returning the value from `explainFlatPlanInJSONFormat` is enough.

Reply:
fixed